UnityXrefMaps/XrefMap.cs

```diff
@@ -12,7 +14,14 @@ namespace DocFxForUnity
     public sealed class XrefMap
     {
         private static readonly Deserializer Deserializer = new Deserializer();
-        private static readonly Serializer Serializer = new Serializer();
+
+        private static readonly Serializer Serializer =
+            Serializer.FromValueSerializer(
+                new SerializerBuilder()
+                    .ConfigureDefaultValuesHandling(DefaultValuesHandling.OmitNull)
+                    .BuildValueSerializer(),
+                EmitterSettings.Default
+            );
```

Comment:
It seems good, I'm just curious here: I haven't seen any null value. Do you have an example?